java/ql/src/semmle/code/java/dataflow/ExternalFlow.qll

```diff
@@ -215,6 +215,10 @@ private predicate sinkModelCsv(string row) {
       "java.net;URL;false;openStream;;;Argument[-1];open-url",
       "java.net.http;HttpRequest;false;newBuilder;;;Argument[0];open-url",
       "java.net.http;HttpRequest$Builder;false;uri;;;Argument[0];open-url",
+      "java.net;URLClassLoader;false;findResource;;;Argument[-1];open-url",
```

Comment:
Would it be simpler to call URLClassLoader's newInstance / constructor the sink and assume at some point it will be used to load a class (i.e., make `"java.net;URLClassLoader;false;URLClassLoader;;;Argument[0];Argument[-1];taint"` a sink, `"java.net;URLClassLoader;false;newInstance;;;Argument[0];ReturnValue;taint" and sink and drop the other 4 URLClassLoader sinks that sink the qualifier)